tests/neg/deriving.scala

```diff
@@ -1,6 +1,6 @@
-import reflect.Generic
+import deriving.MirrorOf
 
-sealed trait A derives Generic // error: cannot take shape, it has anonymous or inaccessible subclasses
```

Comment:
These `Generic` tests appeared to date back to a much older version of `Mirror`. Adjusted for current state, though I could do this in a separate PR if desired.

Reply:
If this still emits an error we should keep it as it is.

Reply:
It would be better to keep this file untouched and add the new version in a `tests/neg/deriving-b.scala` test file.

Reply:
OK I'll restore it. I thought, perhaps erroneously, that these tests were failing due to `reflect.Generic` no longer existing instead of failing due to inability to summon a mirror, dating back to when `Mirror` was called `Generic`. But I might have misunderstood.